src/configs/eslint/eslint.base.js

```diff
@@ -1,15 +1,19 @@
 module.exports = {
     parser: '@typescript-eslint/parser',
-    plugins: ['unicorn', '@typescript-eslint', 'react'],
+    plugins: ['unicorn', '@typescript-eslint', 'react', 'eigenspace'],
     parserOptions: {
         ecmaVersion: 2018,
         sourceType: 'module',
-        ecmaFeatures: {
-            jsx: true
-        },
+        ecmaFeatures: { jsx: true },
         project: 'tsconfig.json'
     },
     rules: {
+        'eigenspace/object-properties-carrying': 'error',
+        'eigenspace/conditions': 'error',
+        'no-multi-spaces': 'error',
+        'no-nested-ternary': 'error',
+        'no-extra-parens': 'error',
```

Comment:
We will get an error in this case in *.styles.js because of autoprettier:
`props => (props.mode === Mode.PRIMARY ? '10px' : '12px')`